src/streamlink/plugins/bongacams.py

```diff
@@ -138,13 +129,18 @@ def _get_streams(self):
                                        self._get_stream_uid(stream_source_info['userData']['username'])),
             # Multiple args with same name not supported.
             # Details: https://github.com/streamlink/streamlink/issues/321
-            "conn" : "S:{username} --conn=S:{access_key} --conn=B:0 --conn=S:{data_key}".format(
+            "conn": "S:{username} --conn=S:{access_key} --conn=B:0 --conn=S:{data_key}".format(
                 username=stream_source_info['userData']['username'],
                 access_key=stream_source_info['localData']['NC_AccessKey'],
                 data_key=stream_source_info['localData']['dataKey']
             )
         }
 
+        subprocess_errorlog_path = self.session.options.options.get('subprocess-errorlog-path')
```

Comment:
Yeah but I'd do `self.session.logger.level > 2` for verbose, so it's normal rtmpdump logging for error/warning levels. But I really don't like hard coding those numbers, in case they change... I'm going to look at the logging stuff over the next few days. 

Do you want to add this to RTMPStream and create a new PR just for that?